docs/features/tcp-health-probe.md

```diff
@@ -35,10 +35,30 @@ public void ConfigureServices(IServiceCollection services)
 
 ## Configuration
 
-To make the TCP health check fully functional, you'll  require some configuration values. 
+The TCP probe allows several additional configuration options.
 
-| Configuration key   | Usage        | Type  | Description                                                         |
-| ------------------- | ------------ | ----- | ------------------------------------------------------------------- |
-| `ARCUS_HEALTH_PORT` | **required** | `int` | The TCP port on which the health report is exposed.                 |
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+    // Add TCP health probe with or whitout extra health checks.
+    services.AddTcpHealthProbes(
```

Comment:
Do we use a default `TcpPortConfigurationKey` or make it a mandatory parameter here? I'd say the latter to be honest.

Reply:
Yep, done.